### DISCUSSION TITLE
Replacing 2FA with password?

### DISCUSSION THREAD

**Discussion by voodlaiz** [Web3Auth/Web3Auth#568] [Help]
I'm currently working on a blockchain project in **Unity** and starting to get familiar with Web3Auth to login users to our game.
I've succesfully managed to log the users on different devices using the 2FA of openlogin (recovery 24 words phrase) however, using the 2FA breaks the flow of my game since the player has to:
	
	- Log in with Google
	- Enter his email to receive the 24 recovery phrase
	- Validate the phrase from the mail received

and then, each time that he logs on another device

	- Log in with Google
	- Search for the email (provided that he didn't delete it)
	- Copy the phrase
	- Paste it in the OpenLogin Window
	
The documentation states that the Recovery share can be obtained via a password (replacing the 2FA recovery phrase) but I can't find any good examples on how to implement this. Could you please take a few minutes to explain in detail how can this be achieved?

**Comment by rinzler99** [comment]
Hey Rodrigo, we don't have this functionality at the moment. 

Thanks for pointing it out! Our team has raised a feature request for the same and we will update the docs once this feature is live.

**Reply by voodlaiz** [reply]
Thanks for the reply
